src/Interpreters/Streaming/CachedBlockMetrics.cpp

```diff
@@ -7,22 +7,26 @@ namespace DB
 {
 namespace Streaming
 {
-void CachedBlockMetrics::serialize(WriteBuffer & wb) const
+void CachedBlockMetrics::serialize(WriteBuffer & wb, VersionType version) const
```

Comment:
Do we need serialize the metrics ? Why

Reply:
Before release new version, current version still is 1, so it is required for CI test on version 1